src/functionalTest/resources/features/common/Base_CPO_Payload_Data.td.json

```diff
@@ -0,0 +1,12 @@
+{
+  "_guid_" : "Base_CPO_Data",
```

Comment:
sorry: pet hate when GUIDs and file names don't match.

Reply:
sure thing